### DISCUSSION TITLE
Not support HTTP/2 protocol?

### DISCUSSION THREAD

**Discussion by WOSHIZHAZHA120** [inertiajs/inertia#950] [Help]
I got a error message "All Inertia requests must receive a valid Inertia response, however a plain JSON response was received." when i use HTTP/2 visit my website.

**Comment by WOSHIZHAZHA120** [comment]
But it can worked in HTTP/1.1 protocol.

**Comment by WOSHIZHAZHA120** [comment]
Image: 
![图片](https://user-images.githubusercontent.com/52521836/138551600-a1953e15-416d-4cb4-9df3-bcac96097f3c.png)

**Comment by dodaydream** [comment]
Have the same problem, it seems in the HTTP/2 protocol, all the headers must be in lower case. This might be the case that Inertia.js cannot match the request headers.

reference:
https://stackoverflow.com/questions/64967098/how-to-get-nginx-to-add-a-header-without-converting-it-to-lowercase

**Reply by dodaydream** [reply]
well, seems that it is not the case, still trying to figure it out...
